### DISCUSSION TITLE
Batch PreCommit  delay and it's impact on miners

### DISCUSSION THREAD

**Discussion by stuberman** [filecoin-project/lotus#6520] [Storage Provider] [answered]
It is not clear to me whether the pre commit batch affects sealing. While precommit commit messages are waiting for release (expiration of "PreCommitBatchWait") are those sectors held from further processing/sealing? As an example is PreCommitBatchWait = 8 hours will PC2 sectors queue up on sealing disks until the timer expires? If so, miners will need to be advised to set the wait timer low enough to not run out of disk capacity and cause worker to idle waiting for timer to expire.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
https://docs.filecoin.io/mine/lotus/miner-configuration/#precommitsectorsbatch

` Note, the current batch won't be sent if MinPreCommitBatch is not reached even any of MaxPreCommitBatch, PreCommitBatchWait or PreCommitBatchSlack is hit.`

miners should be aware what this means to the sealing pipeline ?!?
